test/studies/hpcc/HPL/vass/hpl.hpcc2012.chpl

```diff
@@ -398,30 +398,7 @@ proc DimensionalArr.dsiLocalSlice1((sliceDim1, sliceDim2)) {
       if origScalar(2) then (sliceDim1,)
       else (sliceDim1, sliceDim2);
 
-  // This works but uses the '=>' operator...
-  pragma "no auto destroy"
-  var slice => locAdesc.myStorageArr[r1, r2];
-  pragma "no auto destroy"
-  var result: [(...reindexExpr)] => slice;
-  return result;
-
-  /* This rewrite avoiding the '=>' operator doesn't:
   return locAdesc.myStorageArr[r1, r2].reindex({(...reindexExpr)});
```

Comment:
Might be worth a quick -valgrindexe test run, I believe the 'no auto destroy' pragmas were required in order to prevent use-after-free errors.

Reply:
Looks like it works fine.